### PR TITLE
Pg queries tests

### PR DESCRIPTION
**Pull request by anthonysim**

### CODE REVIEW

**Comment by anthonysim** [comment]
Screen Shot 2021-05-11 at 5 39 29 PM

**Comment by anthonysim** [comment]
Screen Shot 2021-05-11 at 5 39 45 PM

File: __testing__/pgEnrolledQueries.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const axios = require('axios');
+const { enrolledPool } = require('../db/postgres/postgresDB');
+
+// ENROLLED
+(async () => {
```
Comment (vbaobao) [inline __testing__/pgEnrolledQueries.js:5]: Interesting way to execute something with async await immediately. Seems handy.

**Review by vbaobao — APPROVED** [review]
LGTM. Interesting way to separate your POSTGRES and Cassandra files. I like the way you executed your querying tests.

**Pull request merged**